src/hound/vcs/svn_test.go

```diff
@@ -0,0 +1,48 @@
+package vcs
```

Comment:
Thanks for the test, although I think we'd rather have a more holistic approach to testing if we are going to add it. Maybe just pull this out for now until we have figured out what we are going to do here.